packages/flame/lib/src/components/mixins/has_paint.dart

```diff
@@ -83,15 +83,12 @@ mixin HasPaint<T extends Object> on Component
       throw ArgumentError('Opacity needs to be between 0 and 1');
     }
 
-    setColor(
-      getPaint(paintId).color.withValues(alpha: opacity),
-      paintId: paintId,
-    );
+    setColor(getPaint(paintId).color.withOpacity(opacity), paintId: paintId);
```

Comment:
This was easier to read with the trailing comma

Reply:
this is copied as is from main - you can fix on a separate PR!